Tests/test_file_eps.py

```diff
@@ -193,53 +210,49 @@ def _test_readline(self, t, ending):
         assert t.readline().strip("\r\n") == "baz", ending
         assert t.readline().strip("\r\n") == "bif", ending
 
-    def _test_readline_io_psfile(self, test_string, ending):
+    def _test_readline_io_psfile(test_string, ending):
         f = io.BytesIO(test_string.encode("latin-1"))
         t = EpsImagePlugin.PSFile(f)
-        self._test_readline(t, ending)
+        _test_readline(t, ending)
 
-    def _test_readline_file_psfile(self, test_string, ending):
-        f = self.tempfile("temp.txt")
+    def _test_readline_file_psfile(test_string, ending):
+        f = str(tmp_path / "temp.bufr")
```

Comment:
```suggestion
        f = str(tmp_path / "temp.txt")
```
Is there any particular reason why you changed this to '.bufr'? I'm imagining that you just picked it as an abbreviation of 'buffer', but I'm hesistant considering that BUFR is a separate file format that we deal with.

Reply:
That was a copy and paste mistake, thanks!